via/request_tools/__init__.py

```diff
@@ -0,0 +1,2 @@
+from via.request_tools.error_handling import handle_errors
+from via.request_tools.headers import add_request_headers, clean_headers
```

Comment:
Should this be `requests_tools` rather than `request_tools`? I think they're specifically tools for working with the `requests` library

Reply:
Yeah, I'll change that.

Reply:
In fact I think I meant it to already be this as the test directory shows... :facepalm: